apps/sveltekit-example/src/app.html

```
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    %sveltekit.head%
+  </head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)